src/main/java/io/seqera/tower/cli/commands/HealthCheckCmd.java

```diff
@@ -0,0 +1,49 @@
+package io.seqera.tower.cli.commands;
+
+import io.seqera.tower.ApiException;
+import io.seqera.tower.cli.responses.HealthCheckResponse;
+import io.seqera.tower.cli.responses.Response;
+import io.seqera.tower.model.ServiceInfoResponse;
+import picocli.CommandLine;
+
+import java.io.IOException;
+
+@CommandLine.Command(
+        name = "health",
+        description = "Checks system health status"
+)
+public class HealthCheckCmd extends AbstractRootCmd {
+    @Override
+    protected Response exec() throws ApiException, IOException {
+        int connectionCheck = 1;
+        int versionCheck = -1;
+        int credentialsCheck = -1;
+        int requiredApiVersion = 160;
```

Comment:
Add the VersionProvider to get the minimum required API version. It can be accessed like this: `app().spec.versionProvider()`. 
http://github.com/seqeralabs/tower-cli/blob/8562434d079f34e7fe756301b06d57e7c734c526/src/main/java/io/seqera/tower/cli/utils/VersionProvider.java#L7-L15

Reply:
You'll need to add a new `getAPIVersion` method at the `VersionProvider`.

Reply:
Sorry, forgot that one!